File: 프로그래머스/unrated/181854. 배열의 길이에 따라 다른 연산하기/배열의 길이에 따라 다른 연산하기.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> arr, int n) {
    int i = 1;
    if(arr.size() %2 == 1){
        i = 0;
    }
    for(;i<arr.size();i+=2){
        arr[i] += n;        
    }
    return arr;
}

File: 프로그래머스/unrated/181854. 배열의 길이에 따라 다른 연산하기/배열의 길이에 따라 다른 연산하기.java
class Solution {
    public int[] solution(int[] arr, int n) {
        int i = 1;
        if(arr.length %2 == 1){
            i = 0;
        }
        for(;i<arr.length;i+=2){
            arr[i] += n;        
        }
        return arr;
    }
}